### DISCUSSION TITLE
GPU not showing both controllers

### DISCUSSION THREAD

**Discussion by IssacI19** [Rem0o/FanControl.Releases#1870] [General]
I have an RTX 4080 GPU with 3 fans and 2 fan controllers ("GPU Fan 1", and "GPU Fan 2" in Hardware info). 
![image](https://github.com/Rem0o/FanControl.Releases/assets/137911543/d6686f01-0f6a-40fc-b44d-4e205ae3e5e7)

![image](https://github.com/Rem0o/FanControl.Releases/assets/137911543/49bf3d91-89c0-4c02-a87b-990eed4a1ec3)

In Fan Control under the Controls section shows up the 2 controllers. But under the speeds section of Fan Control, there's only one "Fan", and the second one is not showing

![image](https://github.com/Rem0o/FanControl.Releases/assets/137911543/2a9a58da-7846-48d7-a1cb-19fd40f10f13)

The second fan controller seems to work but is not in sync with the other controller. For context "GPU Fan 1" is apparently the left fan and the middle fan. and "GPU Fan 2" is the right fan. Both controllers are linked to the "Fan 1" speed in Fan Control, but the first 2 fans are the ones always spinning first and the third fan spins later. Im assuming the third fan is not using Fan Control at all and using the default fan curve that it came with. Is there any way to fix this and have both Fan speeds RPM sensors show up?

**Comment by Rem0o** [comment]
Can you send in your config json file?

**Reply by IssacI19** [reply]
yea sure.

**Reply by IssacI19** [reply]
[My Config File [GH Message].zip](https://github.com/Rem0o/FanControl.Releases/files/11931862/My.Config.File.GH.Message.zip)

**Reply by Rem0o** [reply]
![image](https://github.com/Rem0o/FanControl.Releases/assets/14118956/2907ea84-8041-41b2-95b1-3c9bbc3fa456)

You named your "Fan 2" GPU fan  -> CPU fan

And matched it to your "control 0"  
![image](https://github.com/Rem0o/FanControl.Releases/assets/14118956/c303d5cf-d556-41a1-8779-e4a910bdb707)

**Reply by IssacI19** [reply]
So I have 2 "CPU Fan" under the speeds RPM section and I have 1 "CPU Fan" in the Controls section
![image](https://github.com/Rem0o/FanControl.Releases/assets/137911543/e1c8cadb-6fc4-4601-8cc3-afa326c185d1)

**Reply by Rem0o** [reply]
One of those "CPU fan" in the speed section is actually your missing RTX 4080 fan.

![image](https://github.com/Rem0o/FanControl.Releases/assets/14118956/10818c60-3fcf-447e-9cdc-800199443ac1)

line 972 in your config file, rename it to whatever you can identify easily, then start the app back up.

**Reply by IssacI19** [reply]
So I changed it in the config file to what you told me. I changed it to "Fan 2 - NVIDIA Geforce RTX 4080". But now it won't let me link it to the speed RPM labeled "Fan 2 - NVIDIA Geforce RTX 4080", but I can on "CPU Fan".

![image](https://github.com/Rem0o/FanControl.Releases/assets/137911543/875a7e48-2631-4b1d-8f0e-a300ea27f64e)

![image](https://github.com/Rem0o/FanControl.Releases/assets/137911543/68fe898e-58b3-4cba-bdec-f820c5490569)

**Comment by Rem0o** [comment]
Right. Now it seems like your pairing is all off.   I would run the automatic pairing process from the assisted setup back again to get it sorted out automatically.
![image](https://github.com/Rem0o/FanControl.Releases/assets/14118956/a2d45346-bdc5-4658-9d0d-3f67634e86f2)

**Reply by IssacI19** [reply]
Ok ill let you know what happens

**Reply by RainerRocks** [reply]
So how'd it go ?
